libcudacxx/include/cuda/__iterator/constant_iterator.h

```diff
@@ -33,7 +34,7 @@
 _CCCL_BEGIN_NAMESPACE_CUDA
 
 //! @brief The \c constant_iterator class represents an iterator in a sequence of repeated values.
-template <class _Tp, class _Index = ::cuda::std::ptrdiff_t>
+template <class _Tp, class _Index = ::cuda::std::ptrdiff_t, class _System = __cccl_any_system_tag>
 class constant_iterator
```

Comment:
Remark: I don't like this change for the reason that until this PR, we did not need any concept of a "system" in libcu++. A system was only a Thrust thing.

Reply:
I hate this patch, the problem is that all the world breaks if we replace `thrust::counting_iterator` with `cuda::counting_iterator` because there will be no `minimal_system` available in many cases